reporter/src/main/kotlin/reporters/AntennaAttributionDocumentReporter.kt

```diff
@@ -238,12 +239,16 @@ class AntennaAttributionDocumentReporter : Reporter {
         return outputFiles
     }
 
-    private fun createCopyrightStatement(resolvedLicense: ResolvedLicenseInfo) =
-        resolvedLicense.flatMapTo(sortedSetOf()) { license ->
+    private fun createCopyrightStatement(resolvedLicense: ResolvedLicenseInfo): String {
```

Comment:
The `Reporter` also gets (optionally) a `CopyrightGarbage` list as input.
How about applying the garbage filter before handing the statements over to the `CopyrightStatementsProcessor` ?

Reply:
As the code uses the `resolvedLicense` copyright garbage was already filtered.